src/main/java/fi/okm/jod/yksilo/dto/profiili/SuosikkiDto.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 The Finnish Ministry of Education and Culture, The Finnish
+ * The Ministry of Economic Affairs and Employment, The Finnish National Agency of
+ * Education (Opetushallitus) and The Finnish Development and Administration centre
+ * for ELY Centres and TE Offices (KEHA).
+ *
+ * Licensed under the EUPL-1.2-or-later.
+ */
+
+package fi.okm.jod.yksilo.dto.profiili;
+
+import fi.okm.jod.yksilo.domain.SuosikkiTyyppi;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+public record SuosikkiDto(
+    @Schema(
+            name = "id",
+            requiredMode = Schema.RequiredMode.NOT_REQUIRED,
+            accessMode = Schema.AccessMode.READ_ONLY)
+        Optional<UUID> id,
+    @Schema(name = "suosionKohdeId", requiredMode = Schema.RequiredMode.REQUIRED) @NotNull
+        UUID suosionKohdeId,
+    @Schema(
```

Comment:
Schema is redundant (notnull already makes the property required and the name (and example) is inferred from the field name and type).

Reply:
Changed.